alf/environments/suite_robotics.py

```diff
@@ -141,14 +209,28 @@ def load(environment_name,
     Returns:
         An AlfEnvironment instance.
     """
-    assert (environment_name.startswith("Fetch")
-            or environment_name.startswith("HandManipulate")), (
-                "This suite only supports OpenAI's Fetch and ShadowHand envs!")
+    assert (
+        environment_name.startswith("Fetch")
+        or environment_name.startswith("HandManipulate")
+        or environment_name.startswith("Ant")
```

Comment:
Not sure if it's appropriate to include multiworld Ant in this suite? Note that "Fetch" and "HandManipulate" were introduced together with very similar implementations.

Reply:
> Not sure if it's appropriate to include multiworld Ant in this suite? Note that "Fetch" and "HandManipulate" were introduced at the same time by OpenAI with very similar implementations.

Reply:
> It's probably Ok? The code here is quite simple. The if conditions mark clearly where Ant is the env.

I mean it's hard for someone else to imagine that metaworld Ant is included in this suite_robotics.py file. The reason  of having "robotics" in the name is because OpenAI Gym originally had "robotics" category which contains Fetch and HandManipulate.